### ISSUE TITLE
Running apt-get install build-essential cmake python-dev exuberant-ctags libclang3.4-dev

### ISSUE THREAD

**Issue by barantaran** [krampstudio/webvim#21]
Looks like hyphen missed in README.md:41 for Ubuntu..

~$ sudo apt-get install build-essential cmake python-dev exuberant-ctags libclang3.4-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package libclang3.4-dev
E: Couldn't find any package by regex 'libclang3.4-dev'

~$ apt-cache search libclang
libclang-3.3-dev - clang library - Development package
libclang-3.4-dev - clang library - Development package
libclang-3.5-dev - clang library - Development package

~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.3 LTS
Release:	14.04
Codename:	trusty

**Comment by krampstudio** [comment]
Thank you for reporting, this was an doc issue.

**Comment by muhajirframe** [comment]
libclang-3.4-dev is not available on Ubuntu 16.04 .

And here's what I found on YouComplete's README.md

```
[Complete this step ONLY if you care about semantic completion support for C-family languages. Otherwise it's not necessary.]

Download the latest version of libclang. Clang is an open-source compiler that can compile C/C++/Objective-C/Objective-C++. The libclang library it provides is used to power the YCM semantic completion engine for those languages. YCM is designed to work with libclang version 3.9 or higher.
```
I guess, we don't need C-family languages, no? Correct me I miss understood here.

**Comment by krampstudio** [comment]
@muhajirframe you're right. I was also using the clang compiler option at some point but it's now removed. I've udpated the doc : https://github.com/vim-dist/webvim/commit/1ae67d6f1c931aeac61d724fadda71fa7fa1d944
